db/migrate/20210907170033_change_venue_reference_in_events.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeVenueReferenceInEvents < ActiveRecord::Migration[6.1]
+  def change
+    remove_reference :events, :venues, foreign_key: true
```

Comment:
was this syntax run by Ricardo?

I'm not sure it works at Database level
I know it does at Model level cause ActiveRecord, but not sure at db level